TeamSix/pacman.py

```diff
@@ -44,13 +48,17 @@ def draw_on_screen(self):
 
     def step(self):
         if self.direction == POSITION_UP:
-            self.pacman_position[1] -= 30
+            if self.pacman_position[1] >= STEP_SIZE:
```

Comment:
Having two if statements right under each other is the same as one with the two statements with an OR between them. It results in cleaner code